src/queue.c

```diff
@@ -835,6 +835,10 @@ _dispatch_root_queues_init_once(void *context DISPATCH_UNUSED)
 	}
 }
 
+#if DISPATCH_USE_THREAD_LOCAL_STORAGE
+	static void _libdispatch_tsd_cleanup(void *ctx); // we need declaration
```

Comment:
this prototype should live in the shims/tsd.h header and the function live in init.c